lib/commons/color/get-text-shadow-colors.js

```diff
@@ -76,23 +98,24 @@ function parseTextShadows(textShadow) {
 	return shadows;
 }
 
-function textShadowColor({ colorStr, offsetX, offsetY, blurRadius }) {
+function textShadowColor({ colorStr, offsetX, offsetY, blurRadius, fontSize }) {
 	if (offsetX > blurRadius || offsetY > blurRadius) {
 		// Shadow is too far removed from the text to impact contrast
 		return new Color(0, 0, 0, 0);
 	}
 
 	const shadowColor = new Color();
 	shadowColor.parseString(colorStr);
-	shadowColor.alpha *= blurRadiusToAlpha(blurRadius);
+	shadowColor.alpha *= blurRadiusToAlpha(blurRadius, fontSize);
 
 	return shadowColor;
 }
 
-function blurRadiusToAlpha(blurRadius) {
+function blurRadiusToAlpha(blurRadius, fontSize) {
 	// This formula is an estimate based on various tests.
 	// Different people test this differently, so opinions may vary.
-	return 3.7 / (blurRadius + 8);
+	const relativeBlur = blurRadius / fontSize;
+	return 0.185 / (relativeBlur + 0.4);
```

Comment:
I used a 20px font when I got these numbers, so I divided the numbers by 20. Block 9 in the text-shadow integration tests handle this.